src/editors/array.js

```diff
@@ -32,11 +32,11 @@ JSONEditor.defaults.editors.array = JSONEditor.AbstractEditor.extend({
     if(this.add_row_button) this.add_row_button.disabled = false;
     if(this.remove_all_rows_button) this.remove_all_rows_button.disabled = false;
     if(this.delete_last_row_button) this.delete_last_row_button.disabled = false;
-    
+
```

Comment:
It looks like the only thing that changed in this whole file was whitespace.  Can you revert this file to keep the pull request clean?